README.md

```diff
@@ -373,12 +376,12 @@ date -u "+0.0.0-%Y%m%d-%H-%M-%S"
 1. In the **Find or create a new tag field**, paste the value from the date command above
 1. Select **Create new tab on publish**
 1. Select **Publish release**.
-   This will cause a github action to start.  Once the github action is complete, move on to
-   the next part which will pull the new dex-operator quay image from https://quay.io/repository/identitatem/dex-operator?tab=tags
+   This will cause a github action to start.  Once the github action is complete, the new dex-operator quay image from https://quay.io/repository/identitatem/dex-operator?tab=tags.  Once the dex-operator quay image is available, move on to
```

Comment:
@chrisahl please take a quick peek at this sentence and tweak it to read correctly. Thanks!